### PR TITLE
srcpkgs: mark broken packages to not waste time of build farm

### PR DESCRIPTION
**Pull request by sgn**

### CODE REVIEW

File: srcpkgs/Uranium/template [hunk 1]
```diff
@@ -15,3 +15,4 @@ license="LGPL-3.0-or-later"
 homepage="https://github.com/Ultimaker/Uranium"
 distfiles="https://github.com/Ultimaker/Uranium/archive/${version}.tar.gz"
 checksum=36896e4807d1ce7040b7ea3cb1288a9e175d55e03e1485b19ba09b407e9515e8
+nocross="python3-scipy nocross"
```
Comment (ericonr) [inline srcpkgs/Uranium/template:18]: Does it make sense to add broken warnings for things that depend on other packages that are already broken? If we manage to cross compile Scipy, someone will have to remove all these nocross warnings too. From my quick testing, trying to cross compile Uranium fails as soon as it starts looking for the `python3-scipy` package.

**Comment by fosslinux** [comment]
Not sure I understand why... as @ericonr said why make dependencies nocross too?

**Comment by sgn** [comment]
On 2020-05-22 16:57:05-0700, fosslinux <notifications@github.com> wrote:
> Not sure I understand why... as @ericonr said why make dependencies nocross too?

The build will stop quicker than depending on its dependencies.
For now, it needs to process all dependencies.

-- 
Danh

**Comment by ericonr** [comment]
But processing the dependencies shouldn't waste too much time, I think. Do you think the inconvenience of manually propagating this sort of info is worth it, then?

**Comment by sgn** [comment]
On 2020-05-25 07:06:28-0700, Érico Nogueira Rolim <notifications@github.com> wrote:
> But processing the dependencies shouldn't waste too much time,
> I think. Do you think the inconvenience of manually propagating this
> sort of info is worth it, then?

I think manually propagating this information will cost us once and
benefits a lot.

Let's take a look at
https://build.voidlinux.org/builders/armv6l-musl_builder/builds/26349/steps/shell_3/logs/stdio

It takes use 7mins for doing nothing :/

-- 
Danh

**Comment by fosslinux** [comment]
Hm, ok, fair enough.

**Comment by travankor** [comment]
Agree with sgn, unless you guys have a better way to solve this.

**Comment by sgn** [comment]
@ericonr Is my answer fine to you?

**Comment by q66** [comment]
A better way to solve this would be to clean things up on the builders. The only reason the builders are trying to build these every time is because it used to be built sometime in the past, and now the builders have it inside their state and attempt to rebuild them every time. But this should not be happening in the first place, the builders should stop trying once they see the dependency tree is not resolvable.

**Comment by sgn** [comment]
Yay, I see the comment in IRC, admin of those build server can fix them instead of this PR.
Closing.

**Comment by ericonr** [comment]
@sgn do you think we could open an issue to track that clean up?

**Comment by sgn** [comment]
On 2020-05-26 17:58:56-0700, Érico Nogueira Rolim <notifications@github.com> wrote:
> @sgn do you think we could open an issue to track that clean up? 

Good idea!
I don't know which project should be open that issue, though.

Either this one or void-infra. I don't know.

-- 
Danh

**Comment by ericonr** [comment]
Most package removal stuff I've seen was in this repo, so I think it would be okay to make an issue here. Could you make it? Otherwise I can grab the package names later and collect them.

**Comment by sgn** [comment]
On 2020-05-28 17:38:47-0700, Érico Nogueira Rolim <notifications@github.com> wrote:
> Most package removal stuff I've seen was in this repo, so I think it
> would be okay to make an issue here. Could you make it? Otherwise
> I can grab the package names later and collect them.

Hi Erico,

I don't have access to GitHub webpage for a moment.

Would you mind to help me create it?
Here's the list of packages, all packages are broken in armv[567], except noted:

* Uranium (nocross)
* WoeUSB
* akonadi-calendar
* akonadi-import-wizard
* akregator
* calendarsupport
* cura (nocross)
* eventviews
* falkon
* firefox-esr-i18n (armv[56] + ppc32)
* firefox-i18n (armv[56] + ppc32)
* flowblade
* incidenceeditor
* kalarm
* kmail-account-wizard
* kmailtransport
* korganizer
* libksieve
* mailcommon
* messagelib
* perl-Cairo (nocross)
* perl-Gtk3-SimpleList (nocross)
* pim-data-exporter
* python3-trimesh (nocross)
* qmk (nocross)
* yarn-bin
* yarn

-- 
Danh

**Comment by ericonr** [comment]
@sgn great, thanks!

**Pull request closed without merging**